workflows/node-memory-hog/workflow.yaml

```diff
@@ -120,6 +122,8 @@ spec:
                 metadata:
                   name: kube-proxy-node-memory-hog-chaos
                   namespace: {{workflow.parameters.adminModeNamespace}}
+                  labels:
+                    context: kube-system_kube-proxy
```

Comment:
Can we use the correct namespace and labels if applicable?. For Infra chaos what can we use in place of appLabel? @ksatchit

Reply:
kube-proxy will reside in kube-system, should not need parameterization IMO